django/apps/registry.py

```diff
@@ -45,8 +46,10 @@ def __init__(self, installed_apps=()):
         # Lock for thread-safe population.
         self._lock = threading.Lock()
 
-        # Pending lookups for lazy relations.
-        self._pending_lookups = {}
+        # Maps ("app_label", "ModelName") tuples to lists of functions to be
```

Comment:
Currently the app registry uses "modelname" (`_meta.model_name`) rather than "ModelName" (`_meta.object_name`) everywhere because model names are treated case-insensitively. I don't think that's a very good design but for consistency I would prefer to keep using it.